UpdateUserHttp/UpdateUser.cs

```diff
@@ -85,25 +109,44 @@ public static async Task<HttpResponseMessage> Run([HttpTrigger(AuthorizationLeve
 
             // Check if userID is passed
             // return BadRequest if not present
-            if (userID != null)
+            if (userID == null)
```

Comment:
When sending a request without a user id, E1BadRequest is what message I receive instead of E0NoUserID.
Maybe we should use  if (string.IsNullOrEmpty(userID)) to check if its null or empty.

Reply:
Added a test for an empty userid `"{\"user\": { \"userID\": \"\" } }"` that expects E0NoUserID and that does fail, with IsNullOrEmpty it passes.